### DISCUSSION TITLE
Prevent Republishing Katas with Open Issues

### DISCUSSION THREAD

**Discussion by Kacarott** [codewars/codewars.com#2598] [Ideas]
If a Kata is ever in draft mode with open issues, there is a very high chance that it was unpublished by a mod or the author themselves, with the intent that the issues be resolved before republishing. With this in mind, there is no reason why it should be possible to (re)publish a kata with those issues still open.

Preventing this would make the mods ability to unpublish betas much more useful, as it forces users to address the issues rather than simply republishing immediately. Users who resolve the issues without fixing anything, just to allow republishing, can simply have their right to resolve comments revoked by a mod.

This suggestion would also enable [this suggestion](https://github.com/codewars/codewars.com/discussions/2600).

**Comment by kazk** [comment]
Sounds good to me.

I added a label `proposal/accepted` for ideas that is worth putting in our roadmap.

**Comment by kazk** [comment]
Draft kata with unresolved issues cannot be published anymore.
